pkg/proto/nbcontract/v1/config.proto

```diff
@@ -95,25 +95,26 @@ message Configuration {
   // ensure_no_dupe_promiscuous_bridge specifies whether to ensure no duplicate promiscuous bridge
   bool ensure_no_dupe_promiscuous_bridge = 45; 
 
-  // custom_search_domain specifies the custom search domain configuration
+  // custom_search_domain specifies the custom search domain configurations
   CustomSearchDomain custom_search_domain = 46;
-  optional CustomLinuxOSConfig custom_linux_os_config = 47;
-  
-  //we do not need these in the final contract, these are consts
-  string cse_helper_filepath = 49;
-  string cse_distro_helper_filepath = 50;
-  string cse_install_filepath = 51;
-  string cse_distro_install_filepath = 52;
-  string cse_config_filepath = 53;
 
-  string dhcpv6_service_filepath = 54;
-  string dhcpv6_config_filepath = 55;
+  // custom_linux_os_config specifies the custom Linux OS configurations including SwapFile, SysCtl configs, etc.
+  CustomLinuxOSConfig custom_linux_os_config = 47;
 
+  // azure_private_registry_server specifies the Azure private registry server
   string azure_private_registry_server = 56;
+
+  // private_egress_proxy_address specifies the private egress proxy address
   string private_egress_proxy_address = 57;
-  FeatureState artifact_streaming_enabled = 58;
-  bool is_kata = 59; // we can calculate this on the VHD, will remove from final contract
-  bool needs_cgroupv2 = 60; // we can calculate this on the VHD, will remove from final contract
+
+  // enable_artifact_streaming specifies whether artifact streaming is enabled or disabled on the VM node
+  bool enable_artifact_streaming = 58;
+
+  // is_kata specifies whether the node is a Kata node
+  bool is_kata = 59; 
+
+  // needs_cgroupv2 specifies whether the node needs cgroupv2
+  bool needs_cgroupv2 = 60;
```

Comment:
Hope this goes away soon ... Should this be made optional to allow for this evolution?

Reply:
I left a comment somewhere I couldn't find now, that we should be able to remove this once we actually work on the go binary. So whether it's optional or not should be fine.

Reply:
It seems to have been on the original line
```go
  bool needs_cgroupv2 = 60; // we can calculate this on the VHD, will remove from final contract
```
Maybe let's keep the comment so that we don't forget?

Reply:
will do